stdlib/Random/src/Random.jl

```diff
@@ -325,6 +325,7 @@ Pick a random element or array of random elements from the set of values specifi
 When only one argument is passed besides the optional `rng` and is a `Tuple`, it is interpreted
 as a collection of values (`S`) and not as `dims`.
 
+Also see the `rand!` function.
```

Comment:
Most of these are a paragraph starting with "See also", and contain links:
```suggestion

See also [`rand!`](@ref) to act in-place, and [`randn`](@ref) for normally distributed numbers.
```